package.json

```diff
@@ -65,6 +65,6 @@
   ],
   "engines": {
     "node": ">=18.0.0",
-    "npm": ">=7.10.0 < 10.0.0"
+    "npm": ">=7.10.0"
```

Comment:
Shouldn't the CI adhere to the minimum supported version of `npm` ? If users who use `npm@v7` experience any problems with xrpl.js, we won't detect it since we are upgrading to `npm@v10`.